_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: "Interviewing, applying and getting your first job in iOS"
+date: 2016-01-30 11:09
+author: orta
+categories: [mobile, ios, juniors, interviewing]
+---
+
+When I was at the beginning of my career, my first developer job application was to a design agency who were doing Mac development too. It was pretty nerve racking to apply for my first programming job, and I came into the interview with no idea what to expect.
+
+Now that I'm both further on in my career, and involved with so many juniors in NYC, I'm meeting with a lot of people who are in the same position and I get a lot of questions asking what they can do to prepare. This post _attempts_ at being a comprehensive collection of recommendations. It is subjective, of course, and strongly based towards my experiences.
```

Comment:
Might be worth doing a s/juniors/junior engineers/. 

I don't know if just using the noun "junior" to describe an entry-level programmer is something that would be immediately obvious to the intended audience. The surrounding context might be enough, but I'm imagining e.g. a current college student saying "but I'm in my senior year of university, not junior!"